src/software/thunderscope/thunderscope_main.py

```diff
@@ -297,7 +302,7 @@ def __async_sim_ticker(tick_rate_ms):
 
         # Launch all binaries
         with Simulator(
-            args.simulator_runtime_dir, args.debug_simulator
+            args.simulator_runtime_dir, args.debug_simulator, args.enable_realism
```

Comment:
Thunderscope crashes because `enable_realism` is currently not a defined argument in thunderscope (python). The argument name should be updated so this works.